### PR TITLE
Add the ApplyBatch command for data migration scenario

### LINKED ISSUE #1982: Add the ApplyBatch command
Write the rocksdb key values converted from redis protocol to kvrocks. This command is mainly used in data migration scenarios to avoid redis protocol conversion overhead. 

We will also provide the `low_priority` option, which uses [Low Priority Write](https://github.com/facebook/rocksdb/wiki/Low-Priority-Write) to avoid having a large impact to user writes.

### PR DESCRIPTION
**Pull request by caipengbo**
This close #1982

### CODE REVIEW

**Comment by caipengbo** [comment]
This test data is not easy to build, I will add tests in the subsequent PR related to data migration.

**Review by git-hulk — DISMISSED** [review]

File: src/commands/cmd_server.cc [hunk 1]
```diff
@@ -1257,6 +1290,7 @@ REDIS_REGISTER_COMMANDS(MakeCmdAttr<CommandAuth>("auth", 2, "read-only ok-loadin
                         MakeCmdAttr<CommandSlaveOf>("slaveof", 3, "read-only exclusive no-script", 0, 0, 0),
                         MakeCmdAttr<CommandStats>("stats", 1, "read-only", 0, 0, 0),
                         MakeCmdAttr<CommandRdb>("rdb", -3, "write exclusive", 0, 0, 0),
-                        MakeCmdAttr<CommandAnalyze>("analyze", -1, "", 0, 0, 0), )
+                        MakeCmdAttr<CommandAnalyze>("analyze", -1, "", 0, 0, 0),
+                        MakeCmdAttr<CommandApplyBatch>("applybatch no-multi", -2, "write", 0, 0, 0), )
```
Comment (git-hulk) [inline src/commands/cmd_server.cc]:
```suggestion
                        MakeCmdAttr<CommandApplyBatch>("applybatch", -2, "write no-multi", 0, 0, 0), )
```

**Comment by git-hulk** [comment]
@caipengbo I can add a basic test case for this.

**Comment by git-hulk** [comment]
@caipengbo I have appended a commit to add the basic test cases, you can take a look.

File: tests/gocase/unit/applybatch/applybatch_test.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	srv := util.StartServer(t, map[string]string{})
+	defer srv.Close()
+
+	ctx := context.Background()
+	rdb := srv.NewClient()
+	defer func() { require.NoError(t, rdb.Close()) }()
+
+	t.Run("Make sure the apply batch command works", func(t *testing.T) {
+		// SET a 1
+		batch, err := hex.DecodeString("04000000000000000100000003013105010D0B5F5F6E616D6573706163656106010000000031")
```
Comment (caipengbo) [inline tests/gocase/unit/applybatch/applybatch_test.go:41]: Is it obtained by dumping the WriteBatch?

Comment (git-hulk) [reply tests/gocase/unit/applybatch/applybatch_test.go:41]: Yes, I printed the batch in https://github.com/apache/kvrocks/blob/unstable/src/cluster/replication.cc#L123

File: src/commands/cmd_server.cc [hunk 3]
```diff
@@ -1222,6 +1222,39 @@ class CommandAnalyze : public Commander {
...
 
+class CommandApplyBatch : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    raw_batch_ = args[1];
+    if (args.size() > 2) {
+      if (args.size() > 3) {
+        return {Status::RedisParseErr, errWrongNumOfArguments};
+      }
+      if (!util::EqualICase(args[2], "low_pri")) {
```
Comment (PragmaTwice) [inline src/commands/cmd_server.cc]: Not sure if LOWPRI is better? As far as I recall, it seems Redis doesn't prefer using underscores in command options.

Comment (caipengbo) [reply src/commands/cmd_server.cc]: Indeed, I will change it.

**Comment by git-hulk** [comment]
@caipengbo Looks good, but need to resolve conflicts.

**Review by PragmaTwice — APPROVED** [review]

**Review by git-hulk — APPROVED** [review]

**Pull request merged**